R/RcppArmadillo.package.skeleton.R

```diff
@@ -37,13 +37,16 @@ RcppArmadillo.package.skeleton <- function(name="anRpackage", list=character(),
     skelFunName <- ifelse(haveKitten, "kitten", "package.skeleton")
     message("\nCalling ", skelFunName, " to create basic package.")
 
+    
+    
+    
```

Comment:
Not sure we need these but that is minor.

Reply:
:) For sure. I'll delete them and amend.